tests/test_evaluator_timeout.py

```
@@ -0,0 +1,459 @@
+"""
+Tests for evaluation timeout functionality in openevolve.evaluator
+"""
+
+import asyncio
+import os
+import tempfile
+import time
+import unittest
+from unittest.mock import patch, MagicMock
```

Ruff (0.11.9)

`unittest.mock.MagicMock` imported but unused

Remove unused import: `unittest.mock.MagicMock`

(F401)